src/tag-input/src/TagInput.js

```diff
@@ -176,6 +176,7 @@ const TagInput = memo(
           size={300}
```

Comment:
What do you think about converting this inner `<Text is "input" />` into something a bit more in line with the way we're going with stuff. Also - can't we bring in `<TextInput />` directly here?

Reply:
Probably could, haven't looked at that. I can do that aswell